Add recovery regex config to docs (#2700)
wso2/docs-is: en/docs/learn/password-recovery.md

        You can edit and customize the email template. For more information
        on how to do this, see [Customizing Automated
        Emails](../../learn/customizing-automated-emails)

> 6. To allow the user's to redirect to a certain location after the password recovery request is completed, change the

Reviewer: ```suggestion
6. To allow the users to be redirected to a certain location after the password recovery request is completed, change the
```

> 6. To allow the users to be redirect to a certain location after the password recovery request is completed, change the

Reviewer 2: ```suggestion
6. To allow the users to be redirected to a certain location after the password recovery request is complete, change the
```

> default configuration in **Recovery callback URL regex** into a specific regular expression which will match with the 

Reviewer: ```suggestion
default configuration in **Recovery callback URL regex** to a specific regular expression that will match the 
```

> redirect URL that the user needs to be redirected.

Reviewer 2: ```suggestion
redirect URL that the user needs to be redirected to.
```

>    redirect URL that the user needs to be redirected.

Reviewer: Is this formatted correctly?

Author: Fixed
